sql/2021/pwa/popular_pwa_libraries_helper.sql

```diff
@@ -0,0 +1,36 @@
+#standardSQL
+# Use this sql to find popular library imports for popular_pwa_libraries.sql
+# And also other importscripts used in service workers
+#
+# TODO - CONVERT TO CUSTOM METRICS - IF WE STILL NEED THIS?
+#
+SELECT
+  client,
+  importscript,
+  COUNT(DISTINCT page) AS pages,
+  total,
+  COUNT(DISTINCT page) / total AS pct
+FROM
+  (SELECT DISTINCT * FROM `httparchive.almanac.service_workers`)
```

Comment:
Same comment to replace `httparchive.almanac.service_workers` with the new script.

Reply:
Yup. These are unticked still. You were going to have a look at the JS function for the ones not ticked yet?